examples/README.md

```diff
@@ -10,19 +10,13 @@
 
 ### How to write and submit a job:
 
-####  (1) Login PAI job webportal
+####  (1) Prepare a job json file
 
-Enter username and password:
+We use CIFAR-10 training job as an OpenPAI job example. 
 
```

Comment:
In this section, we will use **CIFAR-10 training job** as an example to explain how to write and submit a job in OpenPAI.